tron/bin/check_tron_jobs.py

```diff
@@ -57,6 +61,13 @@ def _timestamp_to_timeobj(timestamp):
     return time.strptime(timestamp, '%Y-%m-%d %H:%M:%S')
 
 
+def _are_runs_precious(monitoring_conf):
+    """ Checks if runs are 'precious', that is, we want to ensure there is a
+    successful run per day """
+    precious_jobs_attr = 'check_that_every_day_has_a_successful_run'
```

Comment:
I think this is OK if you are trying to save line length or whatever.
but if it is just a constant, just set it at the top.

Then bonus, the tests can reference it without copy/pasting

Reply:
Sure.